### DISCUSSION TITLE
프로필 이미지 업로드 관련

### DISCUSSION THREAD

**Discussion by Alwysm1le** [storycraft/node-kakao#242] [Q&A] [answered]
일단 염치없는 질문에 죄송합니다.

프로필 사진 변경을 위해서 v3 버전에
uploadLinkImage
uploadLinkPostImage
파일명과 이미지파일버퍼 넣고 함수를 사용하면 400 bad req 에러가 뜨던데
Host 링크가 변경된걸까요?
v4 버전에서는 프로필 사진 변경을 위한 업로드 함수가 없는듯 하여 조심스레 여쭤봅니다..

**Comment by storycraft** [comment, marked as answer]
v3 버전에서 일부 업로드 관련 api에 오류가 있는것으로 보입니다. 프로필 사진 변경을 위한 프로필키는 `file_1`키에 파일을 담은 폼데이터를 https://up-api1-kage.kakao.com/up/open-chat-profile 에  post하시면 됩니다.
